test/DAslabTests/runtests.jl

```diff
@@ -40,5 +40,6 @@ D = 1 / (3 * 20.0)
 
 ## CW
 @test flux_DA_slab_CW(1.0, 0.1, 10.0; n_ext = 1.0, n_med = 1.0, s = 40.0, z = 0.0, xs = 15) ≈ flux_DA_semiinf_CW(1.0, 0.1, 10.0; n_ext = 1.0, n_med = 1.0)
+@test flux_DA_slab_CW(1.0, 0.1, 10.0, n_ext = 1.0, n_med = 1.0, s = 2.0, z = 0.0,xs = 10) = flux_DA_semiinf_CW(1.0, 0.1, 10.0; n_ext = 1.0, n_med = 1.0)
```

Comment:
This looks really great 😄 !!!! 
One thing here for tests is they must be a conditional and evaluate to true or false.
So we could change this to `==` instead.

But we should be careful with strict equality in floating point arithmetic. We are usually only interested if they are approximately equal down to some level. So for this we should use the is approximately `≈` symbol instead which says are these two approxiamtely equal down to some ratio of the floating point epsilon. 

Also, I need to show you how to run the tests locally on your computer!